config/sidekiq_production.yml

```diff
@@ -17,47 +17,47 @@ UpdateCmcPrices:
   cron: '0 * * * * *'   # Runs once per minute
   include_metadata: true
 
-# update_cmc_market_pairs1:
-#   cron: '30 */2 * * * *' # Runs once every other minute
-#   class: UpdateCmcMarketPairs
-#   args: 1
-
-# update_cmc_market_pairs2:
-#   cron: '30 */5 * * * *' # Runs once every 5 minutes
-#   class: UpdateCmcMarketPairs
-#   args: 2
-
-# update_cmc_market_pairs3:
-#   cron: '30 */5 * * * *' # Runs once every 15 minutes
-#   class: UpdateCmcMarketPairs
-#   args: 3
-
-# update_cmc_market_pairs4:
-#   cron: '30 */5 * * * *' # Runs once every 30 minutes
-#   class: UpdateCmcMarketPairs
-#   args: 4
-
-# update_cmc_market_pairs5:
-#   cron: '30 */5 * * * *' # Runs once every 45 minutes
-#   class: UpdateCmcMarketPairs
-#   args: 5
-
-# update_cmc_market_pairs6:
-#   cron: '30 0 * * * *' # Runs once every 1 hour
-#   class: UpdateCmcMarketPairs
-#   args: 6
-
-# update_cmc_market_pairs7:
-#   cron: '30 0 */2 * * *' # Runs once every 2 hours
-#   class: UpdateCmcMarketPairs
-#   args: 7
-
-# update_cmc_market_pairs8:
-#   cron: '30 0 */4 * * *' # Runs once every 4 hours
-#   class: UpdateCmcMarketPairs
-#   args: 8
-
-# update_cmc_market_pairs9:
-#   cron: '30 0 */5 * * *' # Runs once every 5 hours
-#   class: UpdateCmcMarketPairs
-#   args: 9
+update_cmc_market_pairs1:
+  cron: '30 */2 * * * *' # Runs once every other minute
```

Comment:
Hm, wondering if it's worth it to space out the start times by seconds?

Just a bit worried that at the `:30` second mark there will be too many jobs queued simultaneously and it's actually unclear which ones will take precedence.